.github/workflows/registries.yml

```diff
@@ -0,0 +1,67 @@
+name: Registries Tests
```

Comment:
Question for reviewer: I wrote the tests in a GitHub action because initially I was going to run `shellspec` on all OS-es. MacOS does not work in CircleCI. 

But then I realised it's not necessary because we'd be testing the same thing we tests in the contract tests, just with different registries.

So now, if we want, I can move these to CircleCI. Any thoughts? We have a few GitHub actions running and so if that's a problem we could leverage CircleCI